app/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "webpack": "webpack",
-    "watch_webpack": "webpack --watch",
+    "watch": "webpack --watch",
```

Comment:
Just an opportunistic change: I don't think anyone else uses this script other than me; just wanted to shorten the command I have to type.